rules/phpmd.xml

```diff
@@ -59,7 +60,5 @@
         <exclude name="CouplingBetweenObjects" />
         <exclude name="NumberOfChildren" />
     </rule>
-    <rule ref="rulesets/unusedcode.xml">
-        <exclude name="UnusedFormalParameter" />
```

Comment:
Waarom haal je deze weg? We hadden juist gezegd dat dit een standaard regel van ons is omdat je daar met Laravel niet aan ontkomt. Bijv. custom validatieregels die eigenlijk in ieder project zitten introduceren deze al.

Reply:
wat Johan zegt, serieus hoe heb ik die over het hoofd gezien bij het reviewen?  Unused Formal Parameter volgen we niet omdat het alleen maar voor zorgde dat iedereen SuppressWarnings ging toevoegen vanwege interfaces.

Reply:
Ok goed om te weten. Dit is denk ik ook een goed teken dat er centrale coding standards op deze manier komen, want ik heb iets van 6 repositories erbij gehouden om te kijken wat er allemaal geconfigureerd is en de meerderheid had deze rule gewoon aanstaan, dus vandaar dat mijn conclusie was dat deze rule juist niet geexclude moest worden.

Ik zal het aanpassen.

Reply:
Done